ui/src/pages/SiteUsersPage/SiteUserOrgItems.js

```diff
@@ -25,11 +25,37 @@ const enhance = compose(
   withModels
 );
 
-const SiteUserOrgItems = ({ models, user }) => {
-  const orgsItems = models
-    .map(org => <SiteUserOrgItem key={org.get('_id').toString()} org={org} user={user} />)
+const SiteUserOrgItems = ({ models, user, filter }) => {
+  // Decision made via (https://github.com/LearningLocker/learninglocker/pull/1513#issuecomment-587064642)
+  // Need to change when we will have OrganisationUser model.
+  const orgItems = filter
+    .get('_id')
+    .get('$in')
+    .map((org) => {
+      const orgModel = models.find(model =>
+        model.get('_id').toString() === org.get('$oid').toString()
+      );
+
+      if (orgModel === undefined) {
+        return (
+          <li key={org.get('$oid').toString()}>
+            Sorry organisation with id {org.get('$oid')} was deleted!
+          </li>
+        );
+      }
+
+      return <SiteUserOrgItem key={orgModel.get('_id').toString()} org={orgModel} user={user} />;
+    })
     .valueSeq();
-  return <ul>{orgsItems}</ul>;
+
+  const countOfRemainingOrganisations = user.get('organisations').count() - orgItems.count();
+
+  return (
+    <div>
+      <ul>{orgItems}</ul>
+      {countOfRemainingOrganisations > 0 ? <p>Plus { countOfRemainingOrganisations } more not displayed</p> : null}
```

Comment:
It can be reduced
`{countOfRemainingOrganisations > 0 && <p>Plus { countOfRemainingOrganisations } more not displayed</p>}`

Reply:
Done!